### ISSUE TITLE
A11y_AzureToolsStorageExplorer_BlobContainerProeprtiesTab_ScreenReader: VoiceOver not announces State change for "Hide Actions and Properties" control.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1971]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac;  

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Select "Blob Containers" of the selected subscription.
5. Navigate to "Properties" Tab present on left bottom of the screen and hit enter.
6. Start VoiceOver and using Tab key navigate to "Hide Actions and Properties" control and hit enter.
7. Verify State of the control changes to "Show Actions and Properties" but VoiceOver not announces State change of the control.

**​Actual:​**
When hit enter on "Hide Actions and Properties" control state of the control changes to "Show Actions and Properties" but VoiceOver not announces the state change.

**Expected:**
VoiceOver should announce State change for "Hide Actions and Properties" button.

**UserImpact:**
Screen reader users may not be able to know the state change of the control if screen reader not announces state  of the control.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[1971_A11y_AzureToolsStorageExplorer_BlobContainerProeprtiesTab_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669421/1971_A11y_AzureToolsStorageExplorer_BlobContainerProeprtiesTab_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
Only seeing this on macOS. Can confirm that this type of behavior repro's in Chrome also only on macOS: https://bugs.chromium.org/p/chromium/issues/detail?id=1026393

**Comment by MRayermannMSFT** [comment]
Make sure to add to VPAT

**Comment by Alexandria-R** [comment]
@smhigley Can you confirm if this is correct implementation? Ryan asked me to request that.

**Comment by smhigley** [comment]
@MRayermannMSFT it is not expected that a change to the accessible name (whether through `aria-label` or any other means) would necessarily be read out. The right way to solve this would be through a static name, e.g. "Actions and Properties", and an `aria-expanded` attribute that toggles between "true" and "false".

More info here, and a description of the one exception to the "don't dynamically change the name" rule: https://sarahmhigley.com/writing/playing-with-state/

**Comment by MRayermannMSFT** [comment]
Merged a fix for this in. Will create a new build and send to testers soon.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Private build of Mac.
Application version: 1.12.0. 
Now Voice Over announces as "Actions and Properties" control with state Expanded/Collapsed.
This issue is no longer reproduced. Hence Closing the bug.

Added Fixed Video in [One Drive](https://microsoft-my.sharepoint.com/personal/v-nisaho_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fv%2Dnisaho%5Fmicrosoft%5Fcom%2FDocuments%2FStorageExplorer%5FReprovideos&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOmY6L3Avdi1uaXNhaG8vRXNTVUtnZm9PSzlIbUVMOFBxcVhkOTRCVXRBVkJ6S0pldHB5SDZwS0FzVXh5dz9ydGltZT1GRkRuQ25QRzEwZw) for reference.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
